docs/audio.rst

```diff
@@ -3,29 +3,48 @@ Audio
 
 .. py:module:: audio
 
-This module allows you play sounds from a speaker attached to the Microbit.
-In order to use the audio module you will need to provide a sound source.
+This module allows you to play your own sounds. If you are using a micro:bit
+**V2**, ``audio`` is also part of the ``microbit`` module.
 
-A sound source is an iterable (sequence, like list or tuple, or a generator) of
-frames, each of 32 samples.
-The ``audio`` modules plays samples at the rate of 7812.5 samples per second,
-which means that it can reproduce frequencies up to 3.9kHz.
+By default sound output will be via the edge connector on pin 0 and the
+built-in speaker **V2**. You can connect a wired headphones or a speaker to
+pin 0 and GND on the edge connector to hear the sounds.
 
 Functions
 =========
 
 .. py:function:: play(source, wait=True, pin=pin0, return_pin=None)
+                 play(source, wait=True, pin=(pin_speaker, pin0))
```

Comment:
I think V2 will also take the `return_pin` argument without issues, it's just that it ignores it, so we should probably add it:

```suggestion
                 play(source, wait=True, pin=(pin_speaker, pin0), return_pin=None)
```